.github/workflows/lint.yml

```diff
@@ -2,6 +2,9 @@ name: Lint ROS2 package
 on:
   # Run this action whenever a pull request is made
   pull_request:
+  push:
+    branches:
+      - master
```

Comment:
Why add the check for pushing to master now?

Reply:
It wasn't there before when it should have been.
If it's not there, the Workflow badge doesn't accurately display the build status.

Reply:
In future PRs, I think this should be a separate 1-liner PR.

Reply:
Should it be that in this PR?